components/sandbox/tiers/staging/appstudio-env/kustomization.yaml

```diff
@@ -7,6 +7,7 @@ kind: Kustomization
 resources:
 - nstemplatetier-appstudio-env.yaml
 - tiertemplate-appstudio-env-admin-37575401-37575401.yaml
+- tiertemplate-appstudio-env-clusterresources-1854252301-1854252301.yaml
 - tiertemplate-appstudio-env-clusterresources-266544570-266544570.yaml
```

Comment:
@dperaza4dustbit is it safe to remove this older tiertemplate along with these changes?

Reply:
I asked that already in another PR I did and the answer was leave it there for now, they will do clean up eventually